templates/lib/routes.jsx

```diff
@@ -3,6 +3,11 @@ var Router = require('react-router');
 var Route = Router.Route;
 var Link = Router.Link;
 var NotFoundRoute = Router.NotFoundRoute;
+var ga = require('react-ga');
+
+// TODO: this is a dummy GA tracking ID until the real one is ready
+var gaTrackingID = process.env.GA_TRACKING_ID || 'UA-59356678-5';
```

Comment:
You need to add `GA_TRACKING_ID` and `GA_DEBUG` in the webpack.config.js file in the plugin to expose them client side as well.
``` js
  plugins: [
    new webpack.DefinePlugin(importEnvVars([
      'GA_TRACKING_ID',
      'GA_DEBUG'
    ]))
  ]
```